without GNU awk,

awk: weird printf conversion %">                                <r
 input record number 5, file 
 source line number 271

awk: not enough args in printf(<br>                        <hr align="left" width="25%">                        <table width="25%">                                <tr><td><b>Total Jobs</b></td><td  align="center"><b>:</b></td> <td align="right"><b>%'15d</b></td></tr>                                <tr><td><b>Total Files</b></td><td align="center"><b>:</b></td> <td align="right"><b>%'15d</b></td></tr>                                <tr><td><b>Total Bytes</b></td><td align="center"><b>:</b></td> <td align="right"><b>%'15.2f GB</b></td></tr>                        </table>                        <hr align="left" width="25%">)
 input record number 5, file 
 source line number 271


Index: scripts/baculabackupreport.in
--- scripts/baculabackupreport.in.orig
+++ scripts/baculabackupreport.in
@@ -56,7 +56,7 @@ dbtype="@DEFAULT_DB_TYPE@"							# Supported options a
 # dbfile="/catalog/bacula.db" # Uncomment and set to Bacula sqlite3 catalog database file
 db="@db_name@"									# Not needed for sqlite
 dbuser="@db_user@"							# Not needed for sqlite
-dbbin="/usr/bin/psql"
+dbbin="/usr/local/bin/psql"
 # dbpass="-p@db_password@"				# Uncomment and set db password if one is used. Also not needed for sqlite
 
 # Formatting variables
@@ -273,7 +273,7 @@ awkerrfile=$(mktemp -t XXXXXX) 
 # -------------------------
 msg=$(echo "${queryresult}" | \
 LC_ALL=en_US.UTF-8 \
-awk \
+gawk \
 -v html="${html}" \
 -v awkerrfile="${awkerrfile}" \
 -v totaljobs="${totaljobs}" \
